### ISSUE TITLE
Ammo is being rounded up instead of being rounded down

### ISSUE THREAD

**Issue by szefo09** [NekroDarkmoon/ammo-tracker-fvtt#10]
As per D&D5e RAW, half of the ammunition spent should be rounded down, instead of being rounded up. 
This line:
https://github.com/NekroDarkmoon/ammo-tracker-fvtt/blob/04dae4e24581ec1c6316f89acb57d67b3e650450/scripts/tracker.js#L158
Should be:
```js
const recoverable = Math.floor(consumed * (PERCENT * 0.01));
```
Or better yet allow this feature to be selected in settings.

**Comment by NekroDarkmoon** [comment]
Heh, thanks for the report ~ I'll fix it soon. I was hoping to add a setting for that as well as what percentage of ammo can be recovered.

**Comment by NekroDarkmoon** [comment]
Fixed this in 0.2.1
